lib/cassandra/utils/stats/health.rb

```diff
@@ -16,6 +16,27 @@ def metric_name
           'cassandra.service.running'
         end
 
+
+        def gossipstate
+          results = (nodetool_info || '').split("\n")
+          results.map! { |line| line.strip }
```

Comment:
Wondering if `nodetool_info` should be responsibe for the `split` and `strip`?

Reply:
it seems logical that it's only having to do it once.  That was the logic that frank extended over to me.